### PR TITLE
Users/ajya/added task sql dacpac deployment on mgm109

### PR DESCRIPTION
**Pull request by Ajay-MS**

### CODE REVIEW

File: Tasks/SqlDacpacDeploymentOnMachineGroup/task.json [hunk 1]
```diff
@@ -0,0 +1,196 @@
+{
+  "id": "4B506F7F-720F-47BB-BF21-029BAC6A690D",
+  "name": "SqlDacpacDeploymentOnMachineGroup",
+  "friendlyName": "Deploy SQL Dacpac Machine Group(Preview)",
```
Comment (kmkumaran) [inline Tasks/SqlDacpacDeploymentOnMachineGroup/task.json]: Add space before (Preview)

File: Tasks/SqlDacpacDeploymentOnMachineGroup/Main.ps1 [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+$additionalArguments = Get-VstsInput -Name "additionalArguments"
+$additionalArgumentsSql = Get-VstsInput -Name "additionalArgumentsSql"
+
+Import-Module $PSScriptRoot\ps_modules\TaskModuleSqlUtility
+
+Try
+{
+    $connectionString = Escape-SpecialChars -str $connectionString
+    $sqlPassword = Escape-SpecialChars -str $sqlPassword
+    $additionalArguments = Escape-SpecialChars -str $additionalArguments
```
Comment (mvvsubbu) [inline Tasks/SqlDacpacDeploymentOnMachineGroup/Main.ps1]: this is not necessary anymore

Comment (mvvsubbu) [inline Tasks/SqlDacpacDeploymentOnMachineGroup/Main.ps1]: $additionalArguments = Escape-SpecialChars -str $additionalArguments

File: Tasks/SqlDacpacDeploymentOnMachineGroup/Main.ps1 [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+    if ($taskType -eq "dacpac")
+    {
+        Execute-DacpacDeployment -dacpacFile $dacpacFile -targetMethod $targetMethod -serverName $serverName -databaseName $databaseName -authscheme $authscheme -sqlUsername $sqlUsername -sqlPassword $sqlPassword -connectionString $connectionString -publishProfile $publishProfile -additionalArguments $additionalArguments
+    }
+    else
+    {
+        Execute-SqlQueryDeployment -taskType $taskType -sqlFile $sqlFile -inlineSql $inlineSql -serverName $serverName -databaseName $databaseName -authscheme $authscheme -sqlUsername $sqlUsername -sqlPassword $sqlPassword -additionalArguments $additionalArguments
+    }
+
```
Comment (mvvsubbu) [inline Tasks/SqlDacpacDeploymentOnMachineGroup/Main.ps1]: remove empty line

**Pull request merged**